packages/build/src/plugins/resolve.js

```diff
@@ -78,17 +85,24 @@ const resolvePluginPath = async function ({
     return { ...pluginOptions, pluginPath: manualPath, loadedFrom: 'package.json' }
   }
 
-  await getPluginsList({ debug, logs, testOpts })
```

Comment:
This statement was only there to check its reliability. It can be safely moved to its real place.